mod-clans/src/main/java/gg/solarmc/loader/clans/ClanManager.java

```diff
@@ -186,5 +187,56 @@ public void deleteClan(Transaction transaction, Clan clan) {
         clans.invalidate(clan.getClanId());
     }
 
+    /**
+     * Gets x amount of the highest ranked clans by kills
+     * Ordered by highest to lowest.
+     * @param amount the amount of clans to fetch
+     * @return an ordered list (highest first, lowest last) of
+     * clans ranked by stat type.
+     */
+    public List<Integer> getTopClanKills(Transaction transaction, int amount) {
+        Result<Record2<Integer,Integer>> result = transaction.getProperty(DSLContext.class).select(CLANS_CLAN_INFO.CLAN_ID, CLANS_CLAN_INFO.CLAN_KILLS)
```

Comment:
You don't need to select the kills if you're not going to retrieve them. Same with the deaths and assists. You can still sort by the column without pulling it over the network.

Reply:
Oh, my mistake. This should be returning kills as well.